<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinestraGioco</class>
 <widget class="QMainWindow" name="FinestraGioco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Candy Crush!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="AreaGioco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>881</width>
      <height>871</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelPunteggio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Punteggio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPunti">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNome">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMosseRimastetxt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mosse rimanenti:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMosseRimaste">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGiocatore">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Giocatore:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGiocatore_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">Record:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRecord">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string notr="true">n/a</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_new_record">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string notr="true">Nuovo record!!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNuova_partita"/>
    <addaction name="actionMossa_intelligiente"/>
    <addaction name="separator"/>
    <addaction name="actionEsci"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuova_partita">
   <property name="text">
    <string>Nuova partita...</string>
   </property>
  </action>
  <action name="actionEsci">
   <property name="text">
    <string>Esci</string>
   </property>
  </action>
  <action name="actionMossa_intelligiente">
   <property name="text">
    <string>Mossa intelligente...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
